.github/workflows/commit-checker.yaml

```diff
@@ -0,0 +1,46 @@
+name: Unit Tests
```

Comment:
Looks like we have a workflow named `Unit Tests` already, maybe rename this one?